tests/fixtures.py

```diff
@@ -228,9 +228,10 @@ def finalizer():
         """
         Delete multiple PVCs
         """
+        validate_pv_deleted = getattr(class_instance, 'validate_pv_deleted', False)
         if hasattr(class_instance, 'pvc_objs'):
             for pvc_obj in class_instance.pvc_objs:
-                pvc_obj.delete()
+                pvc_obj.delete(validate_pv_deleted=validate_pv_deleted)
```

Comment:
I don't think this is the behavior we want.
We will delete every pv and will wait for it to be deleted instead of deleing all and than wait for all (much faster)